v3/integrations/nrgrpc/go.mod

```diff
@@ -5,10 +5,12 @@ module github.com/newrelic/go-agent/v3/integrations/nrgrpc
 go 1.11
 
 require (
+	github.com/golang/lint v0.0.0-20180702182130-06c8688daad7 // indirect
 	// protobuf v1.3.0 is the earliest version using modules, we use v1.3.1
 	// because all dependencies were removed in this version.
-	github.com/golang/protobuf v1.3.1
+	github.com/golang/protobuf v1.3.3
+	github.com/kisielk/gotool v1.0.0 // indirect
 	github.com/newrelic/go-agent/v3 v3.0.0
```

Comment:
Since we don't have a "replace" directive in the go.mod file, the tests will break on customers' systems. Let's put the Go Agent requirements up to 3.12.0. We'll have better luck in the GitHub Actions CI with that, too.

Reply:
Agreed. I'll make the change and clean up a couple of things and try the pull request again.